src/cpp/plugin/reflection/reflection.proto

```diff
@@ -0,0 +1,93 @@
+// Copyright 2016, Google Inc.
```

Comment:
The proto should go under src/proto directory.

Let's mark it as alpha and check it in first in a separate PR.